js/index.js

```diff
@@ -191,8 +227,49 @@ function populateDeviceTypes(imageConfig) {
     });
 }*/
 
+function populateSupportedDevelopKits(developKitsConfig) {
+    developKitsRadioGroup.innerHTML = "";
+    let i = 1;
```

Comment:
Could we avoid using the variable i since it is not used anywhere in the function?